variables.tf

```diff
@@ -948,3 +948,9 @@ variable "runner_extra_config" {
   type        = string
   default     = ""
 }
+
+variable "enable_user_data_diffing" {
```

Comment:
suggestion: Can we rethink the name? I have no alternative at hand, but `diffing` sounds a little bit strange.

Reply:
WYDT about `show_user_data_in_plan` ?

Reply:
Well, depends on how you look at it. The objective here is to show what changes are actually in those files. To do that I have to write them to somewhere so that we get a diff from `local_file` as shown in the screenshot in the PR body. For me personally writing the file is a clever workaround, but was not the main goal, insofar that if you could not write it, I wouldn't have done it initially.